spec/index.bs

```diff
@@ -1787,10 +1787,10 @@ internal slots of a [=texture=] internal object once we have one.
         defaults to {{GPUTextureViewDimension/"3d"}}.
 
   * {{GPUTextureViewDescriptor/mipLevelCount}}:
-    If 0, defaults to |texture|.{{GPUTextureDescriptor/mipLevelCount}} &minus; {{GPUTextureViewDescriptor/baseMipLevel}}.
+    If undefined or "null", defaults to |texture|.{{GPUTextureDescriptor/mipLevelCount}} &minus; {{GPUTextureViewDescriptor/baseMipLevel}}.
```

Comment:
is `null` actually allowed for optional dictionary fields? I haven't bothered to check....

I usually just say "unspecified" as a catch-all for whatever IDL does.

Reply:
Just curious, what if it's specified as `mipLevelCount: null`?
It's technically specified, but we'd want to treat it as unspecified.

Reply:
Depends on whether that's allowed by the IDL conversion rules (a quick test tells me it probably is).

We should find language from another spec and adopt it. here's a random one I found in the web audio spec:

https://www.w3.org/TR/webaudio/#ref-for-dom-audiocontextoptions-samplerate

> If contextOptions.sampleRate is specified, set the sampleRate of this AudioContext to this value.

Reply:
Here we are operating on the IDL representation rather than the ECMAScript representation. So this is after this algorithm has run:
https://heycam.github.io/webidl/#es-dictionary
but I'm having a hard time understanding this algorithm - it doesn't seem to say anything about how to handle optional members.

Reply:
More specifically, this:

> 3. If esMemberValue is not undefined, then:
>     - Let idlMemberValue be the result of converting esMemberValue to an IDL value whose type is the type member is declared to be of.

seems to say that `null` is NOT supposed to be treated as `undefined` for an optional member, and instead should fail conversion from `null` to `GPUIntegerCoordinate`.

Reply:
Resolved that it doesn't matter:
- If IDL says this is an error, we don't see the `null` value.
- If IDL says this is not an error, the IDL should convert JS `null` into IDL `Undefined`.